common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java

```diff
@@ -31,5 +31,9 @@ public enum SpanName {
     CryptoFactoryEvent,
     SetScopeForDMAgentForFoci,
     GetAccounts,
-    RemoveAccount
+    RemoveAccount,
+    WorkplaceJoin,
```

Comment:
I think we need to know the specific join API individually i.e. `withTokens` vs `preauthorized`. 

Or perhaps you intend to track that through some other means?

Reply:
yes I am tracking it with join_flow_type